helm/production2-values.yaml

```diff
@@ -1,5 +1,5 @@
 environment: production

```

Comment:
I think it would be good to get rid of the production2-values and only have a single production-values.  Originally production related to a cluster on my personal Azure and prod2 was PC4.  Not strictly related to this ticket so can be done in future.  If you do make the changes now, remember to update the release pipeline to run the correct transform

Reply:
Removed prod2